ang/civicase/CaseDetails--placeholder.html

```diff
@@ -194,16 +194,18 @@ <h3><div class="civicase__loading-placeholder__oneline" style="width: 12em;"></d
           </div>
           <div class="panel-body">
             <div class="civicase__activity-feed__list">
-              <div class="civicase__activity-feed__activity-container">
-                <div class="civicase__activity-card civicase__activity-card--long" ng-repeat="i in [0,1,2,3,4]">
-                  <div class="panel panel-default">
-                    <div class="panel-body">
-                      <div class="civicase__activity-card-row" style="margin-bottom: 8px;">
-                        <div class="civicase__loading-placeholder__oneline" style="width: 0.8em; font-size: 24px; margin-right: 10px;"></div>
-                        <div class="civicase__loading-placeholder__oneline" style="width: 8em; font-size: 18px;"></div>
-                        <div class="civicase__activity__right-container civicase__loading-placeholder__oneline" style="width: 7em;"></div>
+              <div class="civicase__activity-feed__list__activity-item">
```

Comment:
Shouldn't this be `civicase__activity-feed__list-item` as it is already inside the activity feed so no need to mention activity again.